src/networkapi.jl

```diff
@@ -179,7 +179,7 @@ coefficient of the ith product within the jth reaction.
 Note:
 - Set sparse=true for a sparse matrix representation
 """
-function prodstoichmat(::Type{SparseMatrixCSC{Int}}, rn::ReactionSystem; smap=speciesmap(rn))
+function prodstoichmat(::Type{SparseMatrixCSC{Int,Int}}, rn::ReactionSystem; smap=speciesmap(rn))
```

Comment:
Is it good idea to be using `SparseMatrixCSC{Int,Int}` entries ? i also remember some conversation thread in Catalyst issues to leave scope for non-integer stoichiometries later  probablt in furture